truss-chains/truss_chains/code_gen.py

```diff
@@ -79,11 +81,8 @@ def _run_simple_subprocess(cmd: str) -> None:
 
 
 def _format_python_file(file_path: pathlib.Path) -> None:
-    _run_simple_subprocess(
-        f"autoflake --in-place --remove-all-unused-imports {file_path}"
-    )
-    _run_simple_subprocess(f"black {file_path}")
-    _run_simple_subprocess(f"isort {file_path}")
+    _run_simple_subprocess(f"ruff check {file_path} --fix --select F401,I")
```

Comment:
Since you're making these changes, could we replace error codes with human readable errors (everywhere)?

Reply:
I can add comments better clarifying for each rule, but Ruff only supports specific codes for rule-sets as of now. https://github.com/astral-sh/ruff/issues/1773.

Reply:
https://github.com/basetenlabs/truss/pull/971/commits/1d744d87b1260206d6dd63dfd85abe5320a5e099